doc/manipulating_images/masker_objects.rst

```diff
@@ -8,10 +8,6 @@ This chapter introduces the maskers: objects that go from
 neuroimaging volumes, on the disk or in memory, to data matrices, eg of
 time series.
 
-.. contents:: **Chapters contents**
```

Comment:
Trying to summarize all of my changes related to just this file in one place:

- We have a mix of admonition types in this file. Several of the `topic` admonitions are a running reference to sklearn, so I think they're OK to keep, but [this one](https://github.com/nilearn/nilearn/pull/3125/files#diff-058459b3f210e511ba10547afb134091ee23ba9205a7dbd19224c95ae8cbec85R205) should probably be a warning since it's advertising an upcoming change (which iirc will actually be a deprecation of int64).
- This [linking is broken](https://github.com/nilearn/nilearn/pull/3125/files#diff-058459b3f210e511ba10547afb134091ee23ba9205a7dbd19224c95ae8cbec85R202) to the `NiftiMasker` class name
- This [scale value](https://github.com/nilearn/nilearn/pull/3125/files#diff-058459b3f210e511ba10547afb134091ee23ba9205a7dbd19224c95ae8cbec85R142) is now making the image look too small with the current CSS. Could we drop or increase it ?

Reply:
Sorry your links don't seem to take me to modified lines (they all lead me to the page of changed files, but not to any file in particular). I know this was a long time ago already, sorry sorry :scream_cat:

Reply:
Not sure if it will link anymore since you pulled in new changes :disappointed:  I won't re-generate all the links unless you have trouble finding the line in this file -- hopefully it's clear at least from the text what I'm referring to !